CMake/ctkMacroBuildQtPlugin.cmake

```diff
@@ -88,7 +88,7 @@ macro(ctkMacroBuildQtPlugin)
   # Wrap
   set(MY_QRC_SRCS "")
   if(CTK_QT_VERSION VERSION_GREATER "4")
-    qt5_wrap_cpp(MY_MOC_CPP ${MY_MOC_SRCS} TARGET ${lib_name} OPTIONS -DHAVE_QT5)
+    qt5_wrap_cpp(MY_MOC_CPP ${MY_MOC_SRCS} OPTIONS -DHAVE_QT5)
```

Comment:
I had an issue with TARGET ${lib_name}, for some reason, it treated TARGET as a file and wanted to generate a moc_TARGET.cpp file, and of course it couldn't find the TARGET file.
When I look at the qt5_wrap_cpp documentation [http://doc.qt.io/qt-5/cmake-manual.html], there does not seem to be any TARGET option supported.